src/file_handler.cpp

```diff
@@ -6,12 +6,13 @@
 #include <functional>
 #include <iomanip>
 #include <queue>
+#include<math.h>
```

Comment:
`#include <cmath>` would be a better choice. Read [this](https://stackoverflow.com/questions/15656434/include-cmath-vs-include-math-h-in-a-c-program).